lettuce-scenario/src/main/java/test/apache/skywalking/testcase/lettuce/controller/LettuceController.java

```diff
@@ -40,18 +48,38 @@
 @PropertySource("classpath:application.properties")
 public class LettuceController {
 
+    private Logger logger = LoggerFactory.getLogger(LettuceController.class);
+
+    private static final OkHttpClient OK_HTTP_CLIENT = new OkHttpClient.Builder().connectTimeout(6, TimeUnit.SECONDS)
+            .readTimeout(6, TimeUnit.SECONDS).writeTimeout(6, TimeUnit.SECONDS).build();
+
     @Value(value = "${redis.host}")
     private String address;
 
     @RequestMapping("/lettuce-case")
     @ResponseBody
-    public String lettuceCase() {
+    public String lettuceCase() throws ExecutionException, InterruptedException {
         RedisClient redisClient = RedisClient.create("redis://" + address + ":6379");
         StatefulRedisConnection<String, String> connection0 = redisClient.connect();
         RedisCommands<String, String> syncCommand = connection0.sync();
         syncCommand.get("key");
 
 
+        RedisAsyncCommands<String, String> asyncCommands0 = connection0.async();
+        AsyncCommand<String, String, String> future = (AsyncCommand<String, String, String>) asyncCommands0.set("key_a", "value_a");
+
+        future.onComplete(s -> {
```

Comment:
If this is async call, the function inside should be run in another thread, then the trace should break if you didn't do anything in plugin.

Reply:
I think so.